### PR TITLE
README improvements

### PR DESCRIPTION
**Pull request by spier**
## Changes

Improving the documentation `README.md`.

I ported the changes that I had already filed on a [previous PR](https://github.com/meltwater/the-orchard/pull/1/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5) over here.

### Are there any breaking changes you are aware of in the PR?

No

## General Checklist

* [ ] Change is tested locally
* [ ] Demo is updated to exercise change (if applicable)
* [ ] [WIP] flag is removed from the title

[1]:https://docs.npmjs.com/about-semantic-versioning
[2]:https://docs.npmjs.com/cli/deprecate
[3]:https://github.com/meltwater/applet-orchard/blob/master/docs/README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Putting the following text anywhere in a file will result in that string being r
 We recommend using a separate source file from your output file to allow for source control to ignore the generated file. The CLI command to accomplish this should look something like:
 
 ```bash
-orchard -i ./sourceHtmlFile.html -o ./outputHtmlFile.html
+orchard -i ./sourceFile.html -o ./outputFile.html
```
Comment (spier) [inline README.md:26]: The file type is sufficiently clear from the extension, so the file name could be shortened to improve readability.

File: README.md [hunk 2]
```diff
@@ -113,7 +113,7 @@ orchard build is:
 
 There are three logging levels set by using the `--logging` flag on the cli:
```
Comment (spier) [inline README.md]: **three** or **four**?

Comment (terodox) [inline README.md]:
```suggestion
There are four logging levels set by using the `--logging` flag on the cli:
```

**Review by terodox — APPROVED** [review]
Thank you for the updates!

**Pull request merged**